src/edit_session/bracket_match.js

```diff
@@ -69,17 +69,21 @@ function BracketMatch() {
      * * one Range if there is only one bracket
      *
      * @param {Point} pos
+     * @param {number|null} side
      * @returns {null|Range[]}
      */
-    this.getMatchingBracketRanges = function(pos) {
+    this.getMatchingBracketRanges = function(pos, side) {
```

Comment:
can we use constant for side variable instead of numbers? Something like SIDE_LEFT = 0, SIDE_RIGHT = 1 so it becomes more clear

Reply:
or maybe we can just make it `isBackwards` instead of side and make it boolean?

Reply:
+1 for using `isBackwards`